Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/file_bugs.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (C) 2021-2022 Apple Inc. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1.  Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+# 2.  Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import os
+import re
+import sys
+
+from .command import Command
+from .pull_request import PullRequest
+
+from webkitbugspy import Tracker, radar
+from webkitcorepy import arguments, run, Terminal
+from webkitscmpy import local, log, remote
+from webkitpy.common.system.user import User
+
+from webkitbugspy.radar import Tracker as RadarTracker
+from webkitbugspy.bugzilla import Tracker as BugzillaTracker
+
+
+class FileBugs(Command):
+    name = 'file-bugs'
+    aliases = ['bugs']
+    help = 'File bugzilla issues for commits without one specified'
+
+    BUGZILLA_PLACEHOLDER = 'Need the bug URL (OOPS!).'
```

Comment:
This is more generic, it's actually just "BUG_PLACEHOLDER" (we may move to GitHub issues at some point)